src/store/model/bestbuy-ca.ts

```diff
@@ -3,8 +3,8 @@ import {Store} from './store';
 export const BestBuyCa: Store = {
 	labels: {
 		inStock: {
-			container: '.x-product-detail-page .addToCartButton:enabled',
-			text: ['Add to Cart']
+			container: '.pricingContainer_25k3c',
```

Comment:
What was broken with the current css selector? I just did a check with chrome developer tools and it seems to be a valid check for in stock items.

Reply:
Sitting in the discord support chat a user was getting false positives because it was displaying "add to cart" in a gray box. It appears they rewrote the websites code (I think) since the original bestbuy-ca.ts was made and now every item (unless the schema.org is different) if it's out of stock it doesn't say, it just turns the button gray as opposed to saying out of stock or add to cart.

Reply:
I noticed some strange-ness about the BestBuy checker as well. But for me it was that the add to cart button initially loads in enabled, some lazy load JavaScript comes in and check stock, then is subsequently disables the button. Problem was that the scraper checked the button BEFORE it was disabled.

Reply:
Almost seems like we need some kind of algo to delay the check of "out of stock" by a env amount. If we delay the button enable status check by say 3 seconds this would be a non-issue.

Reply:
This selector wont work. The `.pricingContainer_####` class is appended with a hash, meaning it's going to be different for every item.